schemas/2019-04-01/Microsoft.VMwareCloudSimple.json

```diff
@@ -475,17 +530,18 @@
               }
             },
             {
-              "$ref": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#/definitions/expression"
+              "$ref": "https://schema.management.azure.com/schemas/common/definitions.json#/definitions/expression"
             }
           ],
           "description": "The list of Virtual VSphere Networks"
         }
       },
       "required": [
         "amountOfRam",
+        "guestOS",
```

Comment:
guestOS and guestOSType are not mandatory anymore,
templateId is mandatory for VM creation

Reply:
I've re-run this to pick up the latest swagger changes - commit https://github.com/Azure/azure-resource-manager-schemas/pull/726/commits/76385836159b8c4e9fdb8f38bdd48abf626ae5f3 shows the diff.

Reply:
thank you, looks perfect